src/messenger_bot.py

```diff
@@ -0,0 +1,52 @@
+import os
+import sys
+import time
+from unittest import skip
```

Comment:
Is unittest module even used in here? I don't think it is, although writing unit tests is always a good practice :)
I think that overall pytest is better than unittest so I would use it instead but of course its up to preference.